src/utils/v2v3/contractLoaders/JBTiered721Delegate.ts

```diff
@@ -0,0 +1,18 @@
+import { Contract } from '@ethersproject/contracts'
+import { SignerOrProvider } from 'models/signerOrProvider'
+
+export const loadJBTiered721DelegateContract = async (
```

Comment:
**question** is this used?

Reply:
Yep, it's used before the `adjustTiers` transaction (`NftDrawer/index:184`)